### DISCUSSION TITLE
How do I use binder variables for version comparison

### DISCUSSION THREAD

**Discussion by sansa20** [wixtoolset/issues#7494] [Questions]
I am using a binder variable to read the version of a dll my installer in installing on the system. However, I am trying to incorporate conditional installation. So how do I read the binder variable as a version for comparison.
Following is what I have tried
```

<?xml version="1.0" encoding="utf-8"?>
<Include>
	<!-- TODO: Put your code here. -->
	<?define OpenTapDllInstallerVersion = "v!(bind.FileVersion.OpenTapVersionFile)"?>
</Include>
```

```
 <MsiPackage Id="PrerequisiteSoftware"
                   InstallCondition="OpenTapDllInstalledVersion &lt; OpenTapDllInstallerVersion"/>
```

The install condition is always being detected as false. I am guessing the issue is somewhere in my pre processor variable assignment but I am unable to figure it out

**Comment by nirbar** [comment]
Assign the version binding to a variable, and compare the two variables
~~~
		<Variable Name="MyVersion" Value="!(bind.packageVersion.MyPackage)" Type="version" Persisted="no"/>
...
		DetectCondition="DetectedVersion AND (DetectedVersion = MyVersion)"
~~~
